--- origsrc/marble-16.08.3/src/apps/marble-kde/CMakeLists.txt	2016-10-21 16:51:21.000000000 -0500
+++ src/marble-16.08.3/src/apps/marble-kde/CMakeLists.txt	2017-02-02 19:07:30.265270800 -0600
@@ -27,7 +27,7 @@ set(marblepart_SRCS marble_part.cpp)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../marble-ui)
 kconfig_add_kcfg_files(marblepart_SRCS settings.kcfgc)
 add_library(marble_part ${marblepart_SRCS})
-set_target_properties(marble_part PROPERTIES PREFIX "${CMAKE_SHARED_LIBRARY_PREFIX}")
+set_target_properties(marble_part PROPERTIES PREFIX "lib")
 target_link_libraries(marble_part
     marbleui
     marblewidget
